libraries/AP_WindVane/AP_WindVane.cpp

```diff
@@ -118,7 +118,7 @@ const AP_Param::GroupInfo AP_WindVane::var_info[] = {
     // @Param: SPEED_TYPE
     // @DisplayName: Wind speed sensor Type
     // @Description: Wind speed sensor type
-    // @Values: 0:None,1:Airspeed library,2:Moden Devices Wind Sensor,3:SITL
```

Comment:
What happened to the SITL backend?

Reply:
the parameters had it at 3 but it was 10 in the enum so i have updated the description to match

Reply:
I think we should probably shorten the string for "3" to just "RPM sensor" and then add extra info to the wiki.

Reply:
fixed,  i will do a PR to update the wiki also